### ISSUE TITLE
Add (fix) option to remember cropping settings per item when working with a folder of videos

### ISSUE THREAD

**Issue by balintlaczko** [balintlaczko/VideoAnalysis#39]
It is already there in the [crop-stuff] below the [umenu] but as of now it always stores the last drawn rectangle with the currently recalled item index - that should be fixed.
This should also exist as an option, alongside with the other, which is applying the same cropping settings for each file in a folder.

**Comment by aleksati** [comment]
In what way do you suggest the [crop-stuff] dim storage system should be fixed or altered? As you say, it does work. The only problem I´ve identified is some scheduling issues (in the cropping section) when exporting which messes with the video dims.

**Comment by balintlaczko** [comment]
Yes, the saving part seems to work now, I remember it didn't look right a month ago, but maybe it was OK already. Now the problem is just the the option itself is not implemented, so cropping switches off with each new video, and also items in the folder which don't yet have a cropping rectangle are not refreshing the cropping display, but those will be relatively easy to implement - hopefully. :)

**Comment by aleksati** [comment]
True! I can try to look more into this during the week.

**Comment by aleksati** [comment]
This is now implemented and working pretty nicely. Refreshes back to "spoof" dim if cropping is not specified on a video, and chooses the LAST drawn cropping rect as dim for all videos if the other option is selected. I disconnected the trigger which turns off the "cropping" box when changing videos in a folder. Do we ever want this box to turn off between videos? We can discuss this later I guess.

However, there might be an issue with videos that don´t have 320x240 dim when exporting. I need you to take a look at it on thursday when we merge, @balintlaczko .
